workflow-templates/devonfw-sync-docs.properties.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "devonfw Workflow",
-    "description": "Devonfw sync docs CI workflow template.",
+    "name": "Sync-docs Workflow",
+    "description": "Update Wiki and devonfw guide - CI workflow template",
```

Comment:
Could you be a little more concrete in the Description?
Something like:
"Update the GitHub Repo wiki and the devonfw guide+website by your asciidoc sources from documentation/ folder on the master branch"